std_msgs/String name

---

nav_msgs/OccupancyGrid RESULT